components/style/values/specified/font.rs

```diff
@@ -200,7 +219,11 @@ impl ToComputedValue for KeywordSize {
         let base_size_px = au_to_int_px(base_size as f32);
         let html_size = self.html_size() as usize;
         if base_size_px >= 9 && base_size_px <= 16 {
-            Au::from_px(FONT_SIZE_MAPPING[(base_size_px - 9) as usize][html_size]).into()
+            if cx.quirks_mode == QuirksMode::Quirks {
```

Comment:
to avoid code repetition something like `let table = if quirks {QUIRKS_TABLE} else {REGULAR_TABLE}` would be nice